gamla/graph.py

```diff
@@ -8,37 +8,59 @@
 from gamla import functional, functional_generic
 
 
+def default_accumulator(
+    queue: Any,
+    seen: Any,
+    current: Any,
+    get_neighbors: Callable,
+    key: Callable = toolz.identity,
+) -> Any:
+    for node in get_neighbors(current):
```

Comment:
I think this line belongs to the bfs logic, this would simplify the signature here